cmd/admin-handlers.go

```diff
@@ -284,6 +285,48 @@ func (a adminAPIHandlers) ServerInfoHandler(w http.ResponseWriter, r *http.Reque
 	writeSuccessResponseJSON(w, jsonBytes)
 }
 
+// ServerDrivesPerfInfo holds informantion about address, performance
+// of all drives on one server. It also reports any errors if encountered
+// while trying to reach this server.
+type ServerDrivesPerfInfo struct {
+	Addr  string             `json:"addr"`
+	Error string             `json:"error,omitempty"`
+	Perf  []disk.Performance `json:"perf"`
+}
+
+// DrivesPerfInfoHandler - GET /minio/admin/v1/health/drive
```

Comment:
Perhaps we should make this a bit more extensible by not having a resource instead have a query param, also /v1/health is an odd endpoint for getting disk performance. We can retro fit it .. or have something like 

/v1/performance?type=drive
/v1/performance?type=network 

??

Reply:
yeah that is good idea, I guess that way it is nicely extensible, changing to this approach

Reply:
done